cmake/modules/platform/toolcfg/openxl.cmake

```diff
@@ -0,0 +1,169 @@
+###############################################################################
+# Copyright IBM Corp. and others 2024
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+###############################################################################
+
+if(OMR_OS_AIX)
+	list(APPEND OMR_PLATFORM_C_COMPILE_OPTIONS -qlanglvl=extended)
+	list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qlanglvl=extended0x)
```

Comment:
These look like xlc options, not openxl options.

Reply:
@midronij Which files are currently being used by AIX for Open XL? Looks like I have this section specific to AIX where the flags aren't updated. I am assuming if the changes for AIX are already merged, they may be coming from somewhere else already. Is it safe to assume this section can be removed, just wanted to confirm.

Reply:
On AIX, OpenXL actually uses the GNU cmake file (`cmake/modules/platform/toolcfg/gnu.cmake`). Generally speaking, clang-based compilers (like OpenXL or xlclang) are considered to be GNU compilers by the config and make files. 

That being the case, I opted to modify the existing GNU cmake files to account for the use of OpenXL on AIX, rather than create new ones for OpenXL. So you should be able to safely remove that section and any others that are specific to AIX. The only thing to be careful of is that the addition of this new file doesn't break the AIX build, since my changes don't take its existence into account.

Reply:
yes, it should be mutually exclusive then, Thanks. I will remove the above section.